File: slimadapter/src/main/java/com/jhj/slimadapter/listener/OnItemLongClickListener.java
package com.jhj.slimadapter.listener;

import android.support.v7.widget.RecyclerView;
import android.view.View;

/**
 * 长按事件
 * Created by jhj on 18-10-11.
 */

public interface OnItemLongClickListener {

    boolean onItemLongClicked(RecyclerView recyclerView, View view, int position);

}

File: slimadapterkt/src/main/java/com/zgdj/slimadapterkt/listener/OnItemLongClickListener.kt
package com.zgdj.slimadapterkt.listener

import android.support.v7.widget.RecyclerView
import android.view.View

/**
 * 长按事件
 * Created by jhj on 18-10-11.
 */

interface OnItemLongClickListener {

    fun onItemLongClicked(recyclerView: RecyclerView, view: View, position: Int): Boolean

}
